package.json

```diff
@@ -21,71 +21,78 @@
-    "@storybook/addon-essentials": "^6.5.16",
-    "@storybook/addon-knobs": "^6.4.0",
-    "@storybook/addon-links": "^6.5.16",
-    "@storybook/addon-storysource": "^6.5.16",
-    "@storybook/addons": "^6.5.16",
-    "@storybook/builder-webpack5": "^6.5.16",
-    "@storybook/manager-webpack5": "^6.5.16",
-    "@storybook/mdx2-csf": "^0.0.3",
-    "@storybook/react": "^6.5.16",
-    "@testing-library/react": "^12.1.4",
+    "@babel/core": "^7.23.6",
+    "@babel/preset-env": "^7.23.6",
+    "@babel/preset-react": "^7.23.3",
+    "@babel/preset-typescript": "^7.23.3",
+    "@emotion/styled": "^11.11.0",
+    "@inrupt/base-rollup-config": "^3.0.2",
+    "@inrupt/eslint-config-base": "^3.0.2",
+    "@inrupt/eslint-config-lib": "^3.0.2",
+    "@inrupt/eslint-config-react": "^3.0.2",
+    "@inrupt/jest-jsdom-polyfills": "^3.0.2",
+    "@mui/material": "^5.15.1",
+    "@storybook/addon-actions": "^7.6.6",
+    "@storybook/addon-docs": "^7.6.6",
+    "@storybook/addon-essentials": "^7.6.6",
+    "@storybook/addon-knobs": "^7.0.2",
+    "@storybook/addon-links": "^7.6.6",
+    "@storybook/addon-mdx-gfm": "^7.6.6",
+    "@storybook/addon-storysource": "^7.6.6",
+    "@storybook/addons": "^7.6.6",
+    "@storybook/cli": "^7.6.6",
+    "@storybook/mdx2-csf": "^1.1.0",
+    "@storybook/react": "^7.6.6",
+    "@storybook/react-webpack5": "^7.6.6",
+    "@testing-library/react": "^14.1.2",
     "@testing-library/react-hooks": "^7.0.2",
     "@testing-library/user-event": "^14.1.0",
-    "@types/jest": "^29.4.0",
-    "@types/node": "^20.1.1",
+    "@types/jest": "^29.5.11",
+    "@types/node": "^20.10.5",
     "@types/react": "^17.0.44",
     "@types/react-table": "^7.7.10",
-    "@typescript-eslint/eslint-plugin": "^5.39.0",
-    "@typescript-eslint/parser": "^5.39.0",
+    "@typescript-eslint/eslint-plugin": "^6.15.0",
+    "@typescript-eslint/parser": "^6.15.0",
     "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.2.4",
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-airbnb-base": "^14.2.1",
-    "eslint-config-prettier": "^8.3.0",
+    "babel-loader": "^9.1.3",
+    "core-js": "^3.34.0",
+    "eslint": "^8.56.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-header": "^3.1.1",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^23.13.2",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.29.4",
-    "eslint-plugin-react-hooks": "^4.4.0",
-    "eslint-plugin-storybook": "^0.6.10",
-    "jest": "^29.4.1",
-    "jest-environment-jsdom": "^29.4.1",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jest": "^27.6.0",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-prettier": "^5.1.1",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-storybook": "^0.6.15",
```

Comment:
I think most of these can be removed

Reply:
https://github.com/inrupt/solid-ui-react/pull/901/commits/52fbddd0b2e6a9f94512c8736690f19dc02a6f6b